burn/Cargo.toml

```diff
@@ -11,12 +11,14 @@ repository = "https://github.com/burn-rs/burn"
 version = "0.9.0"
 
 [features]
-default = ["std", "train"]
+default = ["std"]
 experimental-named-tensor = ["burn-core/experimental-named-tensor"]
 std = [
   "burn-core/std",
 ]
-train = ["std", "burn-train"] # Training requires std
+# Training requires std
+train = ["std", "burn-train"]
+train-cli = ["train", "burn-train/cli"]
```

Comment:
I think `train-cli` should be enabled by default without having to specify `cli`. It is the most common usecase.

Reply:
Can we also document (inline is fine) that explains the difference between train and train-cli?